step.rb

```diff
@@ -26,7 +28,15 @@
     exit 1
 end
 
-url = "https://monitoring-sdk.firebaseapp.com/#{lib_version}/libTrace.a"
+path = lib_version
+
+if ['Xcode 11.5', 'Xcode 11.6', 'Xcode 11.7'].include? xcode_version
```

Comment:
Question: if this statement is not met, shouldn't we notify the user?

Reply:
this is for backward compatibility. the happy case will continue to download the latest file